models/public/human-pose-estimation-3d-0001/model.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  Multi-person 3d human pose estimation model based on "Lightweight OpenPose"
+  <https://arxiv.org/pdf/1811.12004.pdf> and
+  "Single-Shot Multi-Person 3D Pose Estimation From Monocular RGB"
+  <https://arxiv.org/pdf/1712.03453.pdf> papers.
+
+  The model input is a blob that consists of a single image of "1x3x256x448" in
+  BGR order.
+
+  The model outputs three blobs: features_3d, heatmaps, and pafs.
+  The first blob contains coordinates in 3d space, the second one contains
+  keypoint heatmaps and the third is keypoint pairwise relations
+  (part affinity fields).
+task_type: human_pose_estimation
+files:
+  - name: human-pose-estimation-3d-0001.tar.gz
+    sha256: d2b158f07bd2f3d921bde1215829ac99afc7e63868d2d6738b24c7079db54efc
+    size: 18421831
+    source: https://download.01.org/opencv/openvino_training_extensions/models/human_pose_estimation/human-pose-estimation-3d.tar.gz
+postprocessing:
+  - $type: unpack_archive
+    format: gztar
+    file: human-pose-estimation-3d-0001.tar.gz
+framework: pytorch
+conversion_to_onnx_args:
+  - --model-path=$dl_dir
+  - --model-name=PoseEstimationWithMobileNet
+  - --model-params=is_convertible_by_mo=True
+  - --import-module=model
+  - --weights=$dl_dir/human-pose-estimation-3d-0001.pth
+  - --input-shape=1,3,256,448
+  - --input-names=data
+  - --output-names=features,heatmaps,pafs
+  - --output-file=$conv_dir/human-pose-estimation-3d-0001.onnx
+model_optimizer_args:
+  - --input=data
+  - --mean_values=data[128.0,128.0,128.0]
+  - --scale_values=data[255.0,255.0,255.0]
+  - --output=features,heatmaps,pafs
+  - --input_model=$conv_dir/human-pose-estimation-3d-0001.onnx
+license: https://raw.githubusercontent.com/opencv/open_model_zoo/master/LICENSE
```

Comment:
This should point to the OpenVINO Training Extensions license.

Reply:
Why, because it is a public one?

Reply:
Because that's where the model is from.